TACC/Core-CMS-Custom
# bug/WP-741: Registration Form cleanup tasks

## Overview
Addresses bugs found while testing React conversion of registration form
…

## Related

- [WP-741](https://tacc-main.atlassian.net/browse/WP-741)
…

## Changes
- Added path for group validation cases to place validation error messages in correct spots
- Added two hidden form fields (just error message elements in component) to allow Yup to place error messages on checkbox groups
- Added some (non-required) fields to Formik's `initialTouched` prop to allow initial values for said fields to write to db if user does not touch said fields
- Changed write for checkbox values in database util functions for entities and contacts to write from booleans instead of strings coming from form
…

## Testing

1. Go to http://localhost:8000/register/request-to-submit/
2. Ensure validation messages appear on Entity Being Registered -> Number/Code, Plan Types, and Types of Files when no values are given in these groups and correctly clear when at least one field in each group is filled out/checked
3. Submit form and check that record at http://localhost:8000/administration/list-registration-requests/ to ensure all data entered is shown in table and View action

## UI

…
